backend/routes/api.php

```diff
@@ -27,13 +27,15 @@
 
 Route::get('/battles', 'BattlegroundController@getAllBattles');
 
+Route::put('/sell', 'MarketplaceController@sellCards');
```

Comment:
Why does sell take an array of cardIDs and not just a single cardID parameter to sell (like all other function calls)?

Reply:
because in the frontend the user selects one or more cards to sell so the cards are passed to the backend together.  I think battles does it the same way in the contract controller